CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - `UIContainerConfig.userInteractionEventSource` to allow tracking of user interaction events (which toggle the visibility of certain components like the `ControlBar`) on a custom element
 
+### Changed
+- Hidden `adSkipButton` is not constantly updated anymore if a non skippable ad is playing
```

Comment:
`` Avoid unnecessary updating of hidden `AdSkipButton` `` (component names always start with an uppercase letter)